src/main/scala/no/ndla/articleapi/service/ExtractService.scala

```diff
@@ -19,10 +20,17 @@ trait ExtractService {
 
   val extractService: ExtractService
 
-  class ExtractService {
+  class ExtractService extends LazyLogging {
     def getNodeData(nodeId: String): NodeToConvert = {
+      val tagsForNode = tagsService.forContent(nodeId) match {
+        case Failure(e) =>
+          logger.warn("Could not import tags for node {}: {}", nodeId, e.getMessage)
```

Comment:
I denne log-statementen kan du godt ta med hele stacktracen 
`logger.warn(s"Could not import tags for node ${nodeId}", e)`